packages/react/src/component-tests/IcFooter/IcFooter.cy.tsx

```diff
@@ -0,0 +1,431 @@
+            <IcFooterLink href="/">Accessibility</IcFooterLink>
+            <IcFooterLink href="/">Styles</IcFooterLink>
+            <IcFooterLink href="/">Components</IcFooterLink>
+            <IcFooterLink href="/">Patterns</IcFooterLink>
+          </IcFooterLinkGroup>
+          <IcFooterLinkGroup slot="link" groupTitle="Grouped heading">
+            <IcFooterLink href="/">Accessibility</IcFooterLink>
+            <IcFooterLink href="/">Styles</IcFooterLink>
+            <IcFooterLink href="/">Components</IcFooterLink>
+            <IcFooterLink href="/">Patterns</IcFooterLink>
+          </IcFooterLinkGroup>
+          <IcFooterLinkGroup slot="link" groupTitle="Grouped heading">
+            <IcFooterLink href="/">Accessibility</IcFooterLink>
+            <IcFooterLink href="/">Styles</IcFooterLink>
+            <IcFooterLink href="/">Components</IcFooterLink>
+            <IcFooterLink href="/">Patterns</IcFooterLink>
+          </IcFooterLinkGroup>
+        </IcFooter>
+      );
+      cy.viewport(1440, 750);
+      cy.compareSnapshot("groupLinks", DEFAULT_TEST_THRESHOLD + 0.05);
+      cy.checkA11yWithWait();
+    });
+
+    it("render logo links", () => {
+      mount(
+        <IcFooter description="The ICDS is maintained by the Design Practice Team. If you've got a question or want to feedback, please get in touch.">
+          <IcFooterLink slot="link" href="/">
+            Accessibility
+          </IcFooterLink>
+          <IcFooterLink slot="link" href="/">
+            Styles
+          </IcFooterLink>
+          <IcFooterLink slot="link" href="/">
+            Components
+          </IcFooterLink>
+          <IcFooterLink slot="link" href="/">
+            Patterns
+          </IcFooterLink>
+          <div slot="logo">
+            <IcFooterLink href="/">
+              <SlottedSVG
+                xmlns="http://www.w3.org/2000/svg"
+                height="48px"
+                viewBox="0 0 24 24"
+                width="48px"
+                fill="#FFFFFF"
+              >
+                <path
+                  d="M12 6.19L17 10.69V18.5H15V12.5H9V18.5H7V10.69L12 6.19ZM12 3.5L2 12.5H5V20.5H11V14.5H13V20.5H19V12.5H22L12 3.5Z"
+                  fill="currentColor"
+                />
+              </SlottedSVG>
+            </IcFooterLink>
+            <IcFooterLink href="/">
+              <SlottedSVG
+                xmlns="http://www.w3.org/2000/svg"
+                height="48px"
+                viewBox="0 0 24 24"
+                width="48px"
+                fill="#FFFFFF"
+              >
+                <path
+                  d="M12 6.19L17 10.69V18.5H15V12.5H9V18.5H7V10.69L12 6.19ZM12 3.5L2 12.5H5V20.5H11V14.5H13V20.5H19V12.5H22L12 3.5Z"
+                  fill="currentColor"
+                />
+              </SlottedSVG>
+            </IcFooterLink>
+          </div>
+        </IcFooter>
+      );
+      cy.compareSnapshot("groupLinks", DEFAULT_TEST_THRESHOLD + 0.5);
```

Comment:
I think this is meant to show links with icons instead of group links so you might need to generate a new screenshot for this test

Reply:
Does the [Links With Logo](packages/react/cypress-visual-screenshots/baseline/IcFooter.cy.tsx-linksWithLogo.png) test satisfy this comment?

Reply:
Yeah I think it was an issue before because the links with logo test was comparing with the group links screenshot so now there's 2 separate screenshots, it's covered 👍

Reply:
I think this test is Logo with links but by mistake use  same variable name as groupLinks instead of logoWithLinks so change name of cy.compareSnapshot("logoWithLinks", DEFAULT_TEST_THRESHOLD + 0.5)